faq.html

```diff
@@ -171,14 +170,13 @@ <h5><b>Q. My talk is not selected for conference, can I opt for OpenSpace? </b><
           <ul class="faq-answer">
               <li> Yes. You can submit an OpenSpace proposal once we start accepting the same. Keep following to further updates</li>
           </ul>
-        </div>
+        </div> -->
```

Comment:
Can you uncomment this question as well. Early proposals are accepted at cfp@in.pycon.org